examples/image-to-text/run_pipeline.py

```diff
@@ -211,7 +234,7 @@ def main():
         ]
 
     if model_type in ["llava", "idefics2", "llava_next", "mllama", "paligemma"]:
-        processor = AutoProcessor.from_pretrained(args.model_name_or_path)
+        processor = AutoProcessor.from_pretrained(args.model_name_or_path, padding_side="left")
```

Comment:
Is this "padding_slide=left" common to the other "llava" and "llava_next" as well?
Title mentions only "mllama/paligemma/idefices2", but this option is set for all.

Reply:
yes. but we need https://github.com/huggingface/optimum-habana/pull/1674 this fix for llava/llava_next first

Reply:
and then, I will add llava/llava_next to https://github.com/huggingface/optimum-habana/blob/main/examples/image-to-text/run_pipeline.py#L324 as well